plugin/src/main/java/com/playmonumenta/scriptedquests/listeners/PlayerListener.java

```diff
@@ -42,6 +45,20 @@ public PlayerListener(Plugin plugin) {
 	public void playerInteractEvent(PlayerInteractEvent event) {
 		Action action = event.getAction();
 		Player player = event.getPlayer();
+
+
+		//In case this gets fixed, force adventure mode left click block to go through playerAnimationEvent
+		if (player.getGameMode() == GameMode.ADVENTURE && action == Action.LEFT_CLICK_BLOCK) {
+			return;
+		}
+
+		//Disable left click events for the player for the next few ticks
+		player.addScoreboardTag("usedinteractable");
```

Comment:
Should not use a tag here - tags are great for data that is really attached to the player and needs to be saved. Instead use MetadataUtils.happenedRecently(). This means we also don't need a runnable.